css/style.css

```diff
@@ -0,0 +1,205 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  font-family: "Lato", sans-serif;
+}
+
+body {
+  margin-bottom: 20px;
+  padding-bottom: 20px;
+}
+
+a {
+  text-decoration: none;
+}
+
+ul {
+  list-style: none;
+}
+
+#bg-container {
+  padding: 0;
+  margin: 0;
+  max-width: 100%;
+  overflow-x: hidden;
+  background: url(/images/background.jpg) 50% 100% no-repeat;
```

Comment:
- [ ] Your background image doesn't load

when it comes to github pages, you need to specify exactly where the file is for example: if the file is in the same directory as your CSS file (or any other file), you do `./filename` the dot refers to the current directory, if it's in another place, use two dots, which means the parents file, so in your case, the image is in the images folder, and your CSS is in CSS folder, so you do `../images/background.jpg`